foris/locale/de/LC_MESSAGES/foris.po

```diff
@@ -555,14 +557,14 @@ msgid ""
 "password. You can then set up your mobile devices, using the QR code "
 "available next to the form."
 msgstr ""
-"Wenn Sie den Router als Wi-Fi-Zugangspunkt verwenden wollen, dann aktivieren "
-"Sie die Wi-Fi-Option. Geben Sie dann die SSID (Netzwerkname) und ein "
+"Wenn Sie den Router als WLAN-Zugangspunkt verwenden wollen, dann aktivieren "
```

Comment:
Everybody here rather speaks of WLAN (wireless LAN) instead of Wi-Fi. It's a language thing, hence I did the change from Wi-Fi to WLAN.